<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpersDialog</class>
 <widget class="QDialog" name="HelpersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>259</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Helpers Editor</string>
  </property>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>221</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDayRate">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>221</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAdd">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>39</y>
     <width>85</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDelete">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>92</y>
     <width>85</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonQuit">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>145</y>
     <width>85</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditWeekend">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>221</y>
     <width>52</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSubmit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>216</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
